src/routes/Root/Root.tsx

```diff
@@ -120,6 +181,26 @@ const Root = () => {
           login={{ url: `/login?lang=${i18n.language}`, text: 'Login' }}
         />
         <ErrorNote />
+        {megaMenuOpen && (
+          <div className="absolute top[68px] left-0 right-0 max-w-[1092px] m-auto">
+            <MegaMenu
+              footer={
+                'Lorem ipsum dolor sit amet, soleat iracundia eos ea, est in modo vivendo moderatius. Ex duo hinc graeco evertitur, nisl affert vel cu. Ne ius quis repudiare. Ne modo eius corpora mea. Ipsum congue definitiones sed in, an sit unum splendide.'
+              }
+              onClose={() => setMegaMenuOpen(false)}
+            >
+              <SimpleNavigationList title={'Ohjeet'} backgroundClassName="" collapsible={!sm}>
```

Comment:
`backgroundClassName` is optional and has a default value, no need for empty value.